packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2076,15 +2076,18 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   parseMeetingInfo(meetingInfo) {
+    const webexMeetingInfo = meetingInfo.body;
```

Comment:
previous code was checking `meetingInfo && meetingInfo.body` so I think it would be safer to do 
```
const webexMeetingInfo = meetingInfo?.body;
```

Reply:
done